src/services/DhService.ts

```diff
@@ -363,18 +354,19 @@ export abstract class DhService<TDH = unknown, TClient = unknown>
 
       panel.webview.html = this.getPanelHtml(title);
 
-      // TODO: This seems to be subscribing multiple times. Need to see if we
-      // can move it inside of the panel creation block
-      panel.webview.onDidReceiveMessage(({ data }) => {
-        this.handlePanelMessage(
-          data,
-          this.panelRegistry
-            .get(title)!
-            .webview.postMessage.bind(
-              this.panelRegistry.getOrThrow(title).webview
-            )
-        );
-      });
+      // TODO: The postMessage apis will be needed for auth in DHE (vscode-deephaven/issues/76).
+      // Leaving this here commented out for reference, but there are some issues
+      // to work through:
+      // 1. This seems to subscribe multiple times. Should see if can move it
```

Comment:
Yes I think this should definitely be in the creation block above, or you need to dispose of the listener before this gets added again.

Reply:
I updated these comments to be more decisive of what needs to be done.